CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [0.7.3]
+### Added
+- add Prop `position` in tooltip compononent, determines where the tooltip will be display
```

Comment:
how about "`position` prop in `Tooltip`" without add?

Reply:
updated